build.gradle

```diff
@@ -88,7 +88,7 @@ repositories {
 }
 
 checkstyle {
-    toolVersion = '7.2'
+    toolVersion = '8.17'
```

Comment:
does checkstyle need to change versions?

Reply:
Part of the AC was to update the dependencycheck, I think this was confused with checkstyle?

I guess it doesn't hurt to keep this but @amobarak could we please upgrade to the latest `org.owasp.dependencycheck` and run the checks again?

Reply:
I reverted this in the new commit 👍 works fine with 7.2

Reply:
Can we still update org.owasp.dependencycheck @amobarak ?

I seem to remember someone mentioning that updating it may reveal further issues

Reply:
Sure, ill give it a go